worker/dbaccessor/worker_test.go

```diff
@@ -138,7 +137,10 @@ func (s *workerSuite) TestStartupNotExistingNodeThenCluster(c *gc.C) {
 	s.hub.EXPECT().Publish(apiserver.DetailsRequestTopic, gomock.Any()).Return(func() {}, nil)
 
 	w := s.newWorker(c)
-	defer w.Kill()
+	defer func() {
```

Comment:
The Kill vs DirtyKill debate on the last patch becomes moot. We only rely on the deferred calls now.